cmd/describe/machinepool/options.go

```diff
@@ -0,0 +1,40 @@
+package machinepool
+
+import (
+	"fmt"
+
+	"github.com/openshift/rosa/pkg/reporter"
+)
+
+type DescribeMachinepoolUserOptions struct {
+	machinepool string
+}
+
+type DescribeMachinepoolOptions struct {
+	reporter *reporter.Object
+
+	args DescribeMachinepoolUserOptions
+}
+
+func NewDescribeMachinepoolUserOptions() DescribeMachinepoolUserOptions {
+	return DescribeMachinepoolUserOptions{machinepool: ""}
+}
+
+func NewDescribeMachinepoolOptions() (*DescribeMachinepoolOptions, error) {
```

Comment:
@hunterkepley Do we need the `error` on the return signature of this function? It only ever returns `nil`